### DISCUSSION TITLE
Steps for DeDRM and KFX plugins with Kindle Book for Windows 10 PC

### DISCUSSION THREAD

**Discussion by tenxt** [apprenticeharper/DeDRM_tools#1581] [General]
I wanted to show this to everyone since I pieced everything together. The information has scattered instructions and you kind of have to piece all the information together. I hope this helps you all!

Here are instructions in the most paraphrased form. At least how it would of helped me anyways..

**Steps for DeDRM and KFX plugins with  Kindle Book for Windows 10 PC:**
1. Download Kindle version 1.24.3 or less.
- Link here: [https://medium.com/@angeldan1989/the-latest-solution-to-remove-drm-from-kindle-kfx-books-2018-e27d6f71df4f](url)
2. Register/Login to your Amazon account.
3. Buy the Kindle book you want on Amazon. Do not buy the rented version because it won't work.
4. Wait for book to download on Kindle Window PC Program
5. Download calibre
- Link here: [https://calibre-ebook.com/download_windows](url)
6. Download DeDRM from apprenticeharper
- Link here: v6.6.3 [https://github.com/apprenticeharper/DeDRM_tools/releases](url)
7. Unzip file
8. Go in to calibre to add plugins under "Preferences". Click the unzipped DeDRM folder. Choose folder "DeDRM_calibre_plugin" and then click the zip file name "DeDRM_plugin" to add the plugin.
9. Click ok on all and then "apply"
10. Exit calibre and open again.
11. Download the KFX plugin
- Link here: [https://www.mobileread.com/forums/showthread.php?t=291290](url)
12. Go in to calibre to add plugins under "Preferences". Click the zip file "KFX Input" to add the plugin.
13. Click ok on all and then "apply"
14. Exit calibre and open again.
15. Now click add books. Click folder "My Kindle Content." Then click the book you want to add.
16. After the book is added, check the side of calibre to see if there was any change to "Format: azw." If the format changed anything like "Format: azw4" etc. then you have been successful.
17. Now you can convert the book in to any other format you wish.
18. Happy reading.

**Comment by enokoneko** [comment]
This article has been inaccessible before.
https://medium.com/@angeldan1989/the-latest-solution-to-remove-drm-from-kindle-kfx-books-2018-e27d6f71df4f

**Comment by ElleKayEm** [comment]
For KFX format, you need Kindle for PC 1.26 or earlier.  But if you disable KFX downloads, you can decrypt books from even the latest version.  See this post for details:  https://www.mobileread.com/forums/showthread.php?t=283371

I recommend using 64-bit calibre if possible.  Otherwise you may run into memory issues decrypting larger files.  Available here:  https://calibre-ebook.com/download_windows64

If you're using calibre 5.x, the current DeDRM is 7.1.0:  https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v7.1.0

If you're using calibre 4.23 or earlier, use DeDRM 6.8.1: https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v6.8.1

With either one, unzip the main file and load DeDRM_plugin.zip into calibre.  (There isn't a DeDRM_calibre_plugin folder anymore.)

The KFX Input plugin (only needed if dealing with KFX format) can be added in calibre from Preferences > Get plugins to enhance calibre.

Generally, the non-KFX formats are better for conversion.  If you have AZW4 format, I recommend using the KindleUnpack plugin to extract the PDF.  It will also convert AZW3 to EPUB.  (Available in calibre from Preferences > Get plugins to enhance calibre.)

Another import detail is that the books you're trying to decrypt need to have been downloaded with the currently installed Kindle for PC on the same machine and user as calibre with DeDRM.

**Comment by exu-g** [comment]
@tenxt All the links in your post point to "https://github.com/apprenticeharper/DeDRM_tools/discussions/url"  
Could you please fix that, so they lead to the correct links?

**Comment by WPTK** [comment]
Is there an updated way to do this now? None of these methods work in 2024.

**Reply by ElleKayEm** [reply]
Update to DeDRM 10.0.9: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9

**Comment by Roze20** [comment]
The blog was blocked; maybe someone reported it.

Use the DeDRM Plugin v10.0.9 and kfx_input plugin v2.23.0, then download Kindle books via Kindle for PC V2.4.0.
Import the downloaded **azw format files (not the azw.md files)** to the Calibre library and convert them.

**Reply by ElleKayEm** [reply]
You're responding to a year old comment on a 4 year old post.  Not surprising links are broken and info is out-of-date.
Current info here:  https://www.mobileread.com/forums/showthread.php?t=361503
If using Kindle for PC, especially note how to block updates.
